ACKNOWLEDGEMENTS.md

```diff
@@ -2,5 +2,5 @@
 
 - The stylized highway shields are inspired by "[Rebusurance](https://github.com/1ec5/rebusurance/)" by Minh Nguyễn.
 - Initial layer code is derived from Oliver Wipfli's [openmaptiles-starter](https://github.com/wipfli/openmaptiles-starter/) style.
-- The font stack was [packaged by the OpenHistoricalMap project](https://github.com/OpenHistoricalMap/map-styles/).
+- The font stack was [packaged from Google Fonts](https://fonts.google.com/).
```

Comment:
Link to the [Google Fonts attribution page](https://fonts.google.com/attribution) for good measure.